chore: fixed typos (#482)
openedx/docs.openedx.org: source/developers/how-tos/add-sphinx-docs-to-a-repo.rst

      if not os.environ.get('DJANGO_SETTINGS_MODULE'):
         os.environ['DJANGO_SETTINGS_MODULE'] = 'test_utils.test_settings'

> #. Run the build again to make sure you've got the standard logos and footers

Reviewer: ```suggestion
#. Run the build again to make sure you have the standard logos and footers
```

nit: "you've got" is too informal for documentation.
